monitoring/monitoring/generator.go

```diff
@@ -129,6 +129,24 @@ func Generate(logger log.Logger, opts GenerateOptions, dashboards ...*Dashboard)
 		}
 	}
 
+	// Generate additional Prometheus assets
+	if opts.PrometheusDir != "" {
+		customRules, err := customPrometheusRules(opts.InjectLabelMatchers)
+		if err != nil {
+			return errors.Wrap(err, "failed to generate custom rules")
```

Comment:
```suggestion
			return errors.Wrap(err, "Failed to generate custom rules")
```

The tiniest of nits but all other wrapped errors in this file start with capital letters.

Reply:
Thanks - I think they're supposed to start with lowercase, but consistency is probably ideal :)